docs/sources/alerting/alerting-rules/create-grafana-managed-rule.md

```diff
@@ -65,7 +65,8 @@ Define a query to get the data you want to measure and a condition that needs to
 1. Add one or more [expressions][expression-queries].
    a. For each expression, select either **Classic condition** to create a single alert rule, or choose from the **Math**, **Reduce**, and **Resample** options to generate separate alert for each series.
 
-   For details on these options, see [Single and multi dimensional rule]
+   **Note:** When using Prometheus, you can use an instant vector and built-in functions, so you don't need to add additional expressions.
```

Comment:
This should use the note admonition shortcode for consistent styling across our documentation -- https://grafana.com/docs/writers-toolkit/write/shortcodes/#admonition-shortcode.

```suggestion
   {{% admonition type="note" %}}
   When using Prometheus, you can use an instant vector and built-in functions, so you don't need to add additional expressions.
   {{% /admonition %}}
```